nvtabular/io/parquet.py

```diff
@@ -1093,7 +1093,8 @@ def _close_writers(self):
         for writer, fn in zip(self.data_writers, _fns):
             writer.close()
             _path = self.fs.sep.join([str(self.out_dir), fn])
-            self.md_collectors[_path][0].set_file_path(fn)
+            if _path in self.md_collectors:
```

Comment:
is this problematic? Should all _path objects exist in self.md_collectors by design?

Reply:
Do we hit a case where _path isn't in in self.md_collectors?

Reply:
yes some of the test cases in CPU testing hit _path that is not a key in the self.md_collectors

Reply:
in the tensorflow dataloader tests for cpu